examples/power_spectra_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============================
+Power Spectrum of a TimeSeries
+===============================
+
+An example showing how to estimate the power spectrum of a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt 
+from scipy import signal
+import astropy.units as u
+
+import sunpy.timeseries
+from sunpy.data.sample import RHESSI_TIMESERIES
+
+###############################################################################
+# Let's first load a RHESSI TimeSeries from SunPy's sample data.
+# This data contains 9 columns, which are evenly sampled with a time step of 4 
+# seconds. 
+
+ts = sunpy.timeseries.TimeSeries(RHESSI_TIMESERIES)
+
+###############################################################################
+# We now use SciPy's periodogram(`~scipy.signal.periodogram`) to estimate the 
```

Comment:
Looking at the final result, i think we want this to be `SciPy's ~scipy.signal.periodogram`